bittide-instances/src/Bittide/Instances/Pnr/Ethernet.hs

```diff
@@ -180,18 +180,20 @@ vexRiscEthernetTop diffClk cpuReset sgmiiClk inp = (j, u, s)
  where
   (sysClk, sysRst) = clockWizardDifferential diffClk cpuReset
   (j, u, s, _) = vexRiscEthernet sysClk sysRst sgmiiClk inp
+
+-- TODO: Add name annotatoion to `Lvds` before we can use `makeTopEntity
 {-# ANN
   vexRiscEthernetTop
```

Comment:
As far I can see this `vexRiscEthernetTop` isn't used anywhere

Reply:
If it was, the `SGMII_[RT]X_[PN]` port names would be incorrectly (uppercase `_p`/`_n`)